app/renderer/api/materialize.js

```diff
@@ -53,5 +53,9 @@ module.exports = {
     },
     closeModals() {
         $('#config-modal').modal('close');
+    },
+    toggleNavBarMenu() {
+        const elNavBarMenu = $('#navBarMenu')
```

Comment:
to be honest, after having a second look to the docs and the resulting code, I think it was better as $(...).click() the toggle. Sorry for making you change this again